src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,54 @@
+package shop;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CartTest {
+    private static final String CART_NAME = RandomStringUtils.randomAlphabetic(10);
+    private static final String REAL_ITEM_NAME = "KASHKAI";
+    private static final double REAL_ITEM_PRICE = 1000.0;
+    private static final double REAL_ITEM_WEIGHT = 1420;
+    private static final String VIRTUAL_ITEM_NAME = "OpenSUSE";
+    private static final double VIRTUAL_ITEM_PRICE = 0.1;
+    private static final double VIRTUAL_ITEM_SIZE_ON_DISK = 40000;
+    Cart testCart = null;
+
+    @BeforeEach
+    void setUp() {
+        testCart = new Cart(CART_NAME);
+
+        RealItem car = new RealItem();
+        car.setName(REAL_ITEM_NAME);
+        car.setPrice(REAL_ITEM_PRICE);
+        car.setWeight(REAL_ITEM_WEIGHT);
+
+        testCart.addRealItem(car);
+    }
+
+    @Test
+    void deleteRealItem() {
+        RealItem car = new RealItem();
+
+        testCart.deleteRealItem(car);
```

Comment:
It will not delete anything, because you have just created new item. Use existing, which is really in cart